sematic/plugins/metrics_storage/sql/sql_metrics_storage.py

```diff
@@ -174,25 +178,29 @@ def get_aggregated_metrics(
 
         try:
             with db().get_session() as session:
-                query = (
-                    session.query(*select_fields)
-                    .filter(*predicates)
-                    .order_by(MetricValue.metric_time)
```

Comment:
This was the bug that was already present before the 2.0 upgrade: if you tried to order by `metric_time`, but had a group_by that didn't include `metric_time` it would error. For "auto" rollup `metric_time` is not grouped by explicitly, but rather a value extracted from it (which isn't sufficient).